ui/components/multichain/network-manager/index.scss

```diff
@@ -17,6 +17,7 @@
 }
 
 .network-manager__tab-content {
+  padding-top: 16px;
```

Comment:
Adds padding to top of tab content so there is breathing room between the tab header. Matches figma.

<img width="477" height="309" alt="Screenshot 2025-08-22 at 21 02 36" src="https://github.com/user-attachments/assets/bbf23a3c-b5ea-4ccb-9281-b4809ec81a91" />